src/main/java/com/kodilla/ecommercee/controller/UserNotFoundException.java

```diff
@@ -0,0 +1,9 @@
+package com.kodilla.ecommercee.controller;
+
+public class UserNotFoundException extends Exception {
```

Comment:
Co do wyjątków to zastanówmy się. Twój wyjatek jest rzucany w momencie gdy wywołamy endpointa szukającego usera. Jak wiadomo endpoint jest wywoływany w przeglądarce , a gdzie zostanie przekazana informacja że został rzucony wyjatek ? W konsoli . Nikt wtedy nie patrzy do konsoli. Musisz zrobić aby informacja o wyjatku została rzucona jako odpowiedź na requesta i pomoze ci w tym adnotacja @ResponseStatus

tutaj o tej adnotacji : https://www.baeldung.com/spring-response-status
Powinieneś ją umieścić nad klasą coś w takim stylu :
@ResponseStatus(HttpStatus.BAD_REQUEST, reason = "CAN NOT FIND USER")

Reply:
@RequestStatus dodany